### ISSUE TITLE
Hooks & Filters -> Crashing

### ISSUE THREAD

**Issue by pkgFiles** [theos/orion#37]
What are the steps to reproduce this issue?
-------------------------------------------
1. Create a new Tweak and set it's filter to com.apple.UIKit
2. Write this line of code in Tweak.x
`class SBDockHook: ClassHook<SBDockIconListView> { // do stuff here }`
3. Add the Interface in the header file for `SBDockIconListView`
4. Inject the tweak to your iDevice and try to open any kind of Application
5. -> CRASH

What happens?
-------------
As this seems to work with objc, it is an issue with Orion. SpringBoard hooks if filter is set to UIKit will result that all Applications are crashing. The code above is just an example. Any kind of SpringBoard hooks will crash here... 
To sum up: it’ll crash if the class you’re hooking is not present in a process.

What were you expecting to happen?
----------------------------------
Whether the problem will be fixed or there is a way to do things like that without crashes in Orion?

Any logs, error output, etc?
----------------------------
[Log.txt](https://github.com/user-attachments/files/15991512/Log.txt)

Any other comments?
-------------------
…

What versions of software are you using?
----------------------------------------
**Operating System:** iOS 16.0.2

**Toolchain Version:** latest

**SDK Version:** iOS 14.5

**Comment by Luki120** [comment]
Targetting UIKit.. is not something you want to do

**Comment by pkgFiles** [comment]
Thx, I'm aware of such kind of hooks and I don't really like to do it but I don't see another way for this tweak to do all the stuff that it needs, since it has to do with every application... Nvm it's closed now as I look for another solution
